### ISSUE TITLE
SAML: Succesful SLO, followed by hitting the 'back' button results in a 'zombie web page' - you *look* logged in, but if you click anything you get redirected to login

### ISSUE THREAD

**Issue by uberbrady** [grokability/snipe-it#9183]
This may or may not be related to #9182 .

If you invoke an SLO from Snipe-IT, it will correctly hit the IdP, and the IdP's response seems to correctly hit Snipe-IT and invalidate the user's session. But after that, the user can hit 'back', and they see a 'normal-looking' Snipe-IT page (whichever page they tried to invoke the logout from).

I'm not sure if it's possible, but I would prefer it if the 'back' button forced the user straight to a 'login' page (which would almost certainly be another redirect to their IdP's login page).

**Comment by johnson-yi** [comment]
Hi uberbrady

Have you tested this behaviour with normal basic auth flow with login/logout? I can reproduce this on the [demo](https://demo.snipeitapp.com/login) site.

**Comment by uberbrady** [comment]
No, not particularly. That does certainly sound like just a plain old regular website problem - and, potentially, one that can't be fixed - 'back' is 'back' and we can't change that unless we do some kind of weirdo socket thingee, and I don't think we'd want to do that. I'll play around with it and possibly close it out if it makes sense to do so.

**Comment by snipe** [comment]
I believe this was resolved in October 2021 via the `PreventBackHistory` middleware
